capstone-ui/src/components/LogOutView/LogOutView.jsx

```diff
@@ -0,0 +1,13 @@
+import * as React from "react";
+import Login from "../Login/Login";
+import SignUp from "../SignUp/SignUp";
+import "./LoggedOutView";
+
+export default function LoggedOutView({ handleLogin }) {
```

Comment:
This component is not being used anywhere yet. Am I correct?
Where are you planning to add it?